test/test_helper.rb

```diff
@@ -5,11 +5,17 @@
 require 'ar_transaction_changes'
 require 'minitest/autorun'
 
+MiniTest::Test = MiniTest::Unit::TestCase unless defined?(MiniTest::Test)
```

Comment:
It is now but since Rails 4 lock a version using MiniTest::Unit::TestCase this is sadly necessary :(